### PR TITLE
Add support for arbitrary parameters in filterIntensity (issue #164)

### PR DESCRIPTION
**Pull request by jorainer**
This PR mainly adds support for passing arguments to the filter function of `filterIntensity` (issue #164).

### CODE REVIEW

File: tests/testthat/test_MsBackendDataFrame.R [hunk 1]
```diff
@@ -505,7 +505,7 @@ test_that("spectraData, spectraData<-, MsBackendDataFrame works", {
 
 test_that("show,MsBackendDataFrame works", {
     be <- MsBackendDataFrame()
-    show(be)
+    expect_output(show(be), "MsBackendDataFrame")
```
Comment (sgibb) [inline tests/testthat/test_MsBackendDataFrame.R:508]: :+1: for testing `show` but aren't the following two lines useless?

**Review by sgibb — APPROVED** [review]
Great, thanks!

**Review by lgatto — APPROVED** [review]

File: tests/testthat/test_MsBackendDataFrame.R (hunk 1 above)
Comment (jorainer) [reply tests/testthat/test_MsBackendDataFrame.R:508]: I will remove them.

**Comment by jorainer** [comment]
Note: I'm just also trying to fix the GH

**Pull request merged**